lib.php

```diff
@@ -4,30 +4,19 @@
 defined('MOODLE_INTERNAL') || die();
 class enrol_payment_plugin extends enrol_plugin {
 
-    public function enrol_page_hook(stdClass $instance){  
-        global $CFG, $USER, $OUTPUT, $PAGE, $DB;
+     public function enrol_page_hook(stdClass $instance){  
+         global $CFG, $USER, $OUTPUT, $PAGE, $DB;
 
-        //saves everything about to happen
         ob_start();
-        echo '<p> this works <p>';
-
-        //Check the price of the course 
-        if (abs($cost) < 0.01) { 
-            //returns "This course is free."
-            echo '<p>'.get_string('nocost', 'enrol_payment').'</p>';
-        }
-
-        
         
-            
-        //stops saving 
-        //ob_end_clean();
+        echo "<p><b>To enrol into a class please pay the amount below</b></p>";
```

Comment:
Might be better to use get_string for language support

Reply:
Also all of these HTML tags should be escaped using \html_writer::tag('p', $string);